### PR TITLE
KAIZ-86: API Development - Service layer functions for Pod Endpoint

### PR DESCRIPTION
**Pull request by LPZ-Fabian**
New Changes:

- Added service layer functions for Pods (fetch, fetch all, delete, and update)
- Created test file in /pages directory to demo the use of the service layer functions for pods.

### CODE REVIEW

File: frontend/services/podService.ts
Comment (VictorSeverin) [inline frontend/services/podService.ts:1]: you're using await when returning data in the last 2 functions but not in the first 2

**Review by VictorSeverin — APPROVED** [review]
LGTM

File: frontend/services/podService.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+		return await response
+	} catch (error) {
+		console.error('Error:', error)
+		throw error
+	}
+}
+
+// Updates a pod using the API
+export async function updatePod(id: string, updateInfo: any) {
```
Comment (ddenaga) [inline frontend/services/podService.ts:49]: Add specific parameters to this function. Should be able to pass in a JSON object `{name, leader, meetingTime, members, meetings}` with each field as optional.

**Review by ddenaga — CHANGES_REQUESTED** [review]

File: frontend/services/podService.ts
Comment (ddenaga) [reply frontend/services/podService.ts:1]: @LPZ-Fabian `await` seems redundant in those last two functions. Use `await` before you call any async function or on a Promise.

**Review by ddenaga — DISMISSED** [review]

File: frontend/services/podService.ts
Comment (LPZ-Fabian) [reply frontend/services/podService.ts:1]: await is necessary when calling both fetchPod() and fetchAllPods() as they are both async functions. Taking this out would not allow for the response to be logged properly. testService() however, has been taken out of onMounted() to fix the redundancy.

**Pull request merged**